code/modules/awaymissions/mission_code/UO71-terrorspiders.dm

```diff
@@ -0,0 +1,257 @@
+
```

Comment:
Do something with this whitespace please.
Get rid of it, add a comment, set it on fire, it just shouldn't be right here.